src_c/mask.c

```diff
@@ -1708,6 +1708,72 @@ mask_init(PyObject *self, PyObject *args, PyObject *kwargs)
     return 0;
 }
 
+typedef struct {
+    int numbufs;
+    Py_ssize_t shape[2];
+    Py_ssize_t strides[2];
+} mask_bufinfo;
+
+static int
+pgMask_GetBuffer(pgMaskObject *self, Py_buffer *view, int flags)
+{
+    bitmask_t *m = self->mask;
+    mask_bufinfo *bufinfo = (mask_bufinfo*)self->bufdata;
+
+    if (bufinfo == NULL) {
+        bufinfo = PyMem_Malloc(sizeof(mask_bufinfo));
```

Comment:
I think there should be an error check here for PyMem_Malloc... something like this?
```c
if (bufinfo == NULL)
    return -1;
```